Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -95,24 +97,13 @@ static ShardMapUtils()
 
         /// <summary>
         /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.
+        /// Below call will throw if connection string specifies Active Directory authentication and library
+        /// is not running with .NET 4.6.
```

Comment:
You are right. Removing this method and adding a devnote. Also removed unused IsActiveDirectoryAuthenticationSupported from ShardMapUtils.cs